### PR TITLE
Fix evictor type for Alluxio cache

### PR DESCRIPTION
**Pull request by apc999**
Test plan - unit tests

Fix evictor policy type for Alluxio cache and make sure the default config is practised in unit tests

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-cache/src/main/java/com/facebook/presto/cache/alluxio/EvictionPolicy.java [hunk 1]
```diff
@@ -16,8 +16,8 @@
 public enum EvictionPolicy
 {
     FIFO("alluxio.client.file.cache.evictor.FIFOEvictor"),
```
Comment (shixuan-fan) [inline presto-cache/src/main/java/com/facebook/presto/cache/alluxio/EvictionPolicy.java:18]: Do we need to change this as well? I see others all have "Cache" 😂

**Review by shixuan-fan — COMMENTED** [review]
Is it expected that we have a lot removal on the tests?

**Review by shixuan-fan — APPROVED** [review]

File: presto-cache/src/main/java/com/facebook/presto/cache/alluxio/EvictionPolicy.java (hunk 1 above)
Comment (apc999) [reply presto-cache/src/main/java/com/facebook/presto/cache/alluxio/EvictionPolicy.java:18]: this one is really named as `FIFOEvictor` , that's why I thought others are named in the same way. My bad

**Comment by apc999** [comment]
> Is it expected that we have a lot removal on the tests?

We are not removing code but instead using the production code path to set unit test.
In this way, we will catch errors like this earlier.

```
AlluxioCachingConfigurationProvider provider = new AlluxioCachingConfigurationProvider(cacheConfig, alluxioCacheConfig);
```

**Pull request merged**